VideoLocker/src/test/java/org/edx/mobile/login/LoginActivityTest.java

```diff
@@ -0,0 +1,53 @@
+package org.edx.mobile.login;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.RuntimeApplication;
+import org.edx.mobile.view.LoginActivity;
+import org.edx.mobile.view.PresenterActivityTest;
+import org.edx.mobile.view.login.LoginPresenter;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.android.api.Assertions.assertThat;
+
+
+/**
+ * Created by cleeedx on 5/20/16.
+ */
```

Comment:
We don't need these comments. You may want to change your IDE settings to stop adding them by default.